examples/src/ref_finance.rs

```diff
@@ -198,7 +198,7 @@ async fn create_custom_ft(
     ft.call("new_default_meta")
         .args_json(json!({
             "owner_id": owner.id(),
-            "total_supply": parse_near!("1,000,000,000 N").to_string(),
+            "total_supply": NearToken::from_near(1_000_000_000).as_yoctonear().to_string(),
```

Comment:
With `serde` feature enabled for near-token, this should just work:

```suggestion
            "total_supply": NearToken::from_near(1_000_000_000),
```